modules/@apostrophecms/modal/ui/apos/mixins/AposEditorMixin.js

```diff
@@ -251,11 +251,18 @@ export default {
             result = false;
             break;
           }
-          if (Array.isArray(self.getFieldValue(key))) {
-            result = self.getFieldValue(key).includes(val);
+
+          const fieldValue = self.getFieldValue(key);
+          const fieldValueOrDef = fieldValue !== undefined
+            ? fieldValue
+            : self.schema.find(field => field.name === key).def;
```

Comment:
use the default values defined in the schema fields to fix conditional fields not showing up when editing existing docs, in the case where new fields have been added but not (yet) saved to the document